src/Main.elm

```diff
@@ -61,6 +61,19 @@ executeAdd model =
     }
 
 
+showResults : Float -> String
+showResults float =
+    let
+        results =
+            String.fromFloat float
+    in
+    if String.length results > 10 then
+        "ERROR"
```

Comment:
If you want this to display ONLY when there are more then 10 digits, then you will need to check for the decimal point here.

Reply:
taken care of, thanks.